plugins/copr.py

```diff
@@ -255,9 +262,10 @@ def run(self):
 
 Please do not file bug reports about these packages in Fedora
 Bugzilla. In case of problems, contact the owner of this repository.
-
-Do you really want to enable {0}?""".format('/'.join([self.copr_hostname,
-                                                      copr_username, copr_projectname])))
+"""))
+            project = '/'.join([self.copr_hostname, copr_username,
+                                copr_projectname])
```

Comment:
Nitpick: this could go on a single line.

Reply:
Not within 80 characters.

Reply:
Our limit is 100.

Reply:
Is it wrong to wrap it if I consider limit 100 to be a wrong limit?

Reply:
Hard to believe, you really like 80? You can keep it in the patch, I don't mind too much. Though it would be better if you adhered to the project's coding style I think :sunglasses: